lib/loader.js

```diff
@@ -124,7 +124,11 @@ Loader.prototype.prepFixtureData = function(data, Model) {
                 promises.push(
                     (typeof val === 'object' ?  assoc.target.find(options) : assoc.target.findOne(options))
                     .then(function(obj) {
-                        result[assoc.identifier] = obj[assoc.target.primaryKeyField || 'id'];
+                        if (obj) {
+                            result[assoc.identifier] = obj[assoc.target.primaryKeyField || 'id'];
+                        } else {
```

Comment:
Can we add a check here to ensure that on-the-fly association creation is what is intended? Actually this area of code currently has bad error reporting.

I suggest we add a check if the option was specified, if not, it should error and say association not found , such:

```
throw new Error('No associated model found for: \n\t' + JSON.stringify(options) + '\nwhile processing fixture: \n\t' + JSON.stringify(data));
```

I'm not positive about the logic to check if your include option was specified, could you add it along with this fallback error case?